roles/kubernetes/control-plane/templates/kubeadm-config.v1beta4.yaml.j2

```diff
@@ -477,6 +477,28 @@ scheduler:
 {% endfor %}
 {% endif %}
 ---
+apiVersion: kubeadm.k8s.io/v1beta4
+kind: UpgradeConfiguration
+apply:
+  kubernetesVersion: {{ kube_version }}
+  allowExperimentalUpgrades: true
+  allowRCUpgrades: true
```

Comment:
nit: I don't think we ever supported rc version so this could probably be dropped

Reply:
I figured it was worth including, if not a bit redundant, since we already enable `allowExperimentalUpgrades` which allows RC + alpha, etc.

> allowExperimentalUpgrades
> bool	
> allowExperimentalUpgrades instructs kubeadm to show unstable versions of Kubernetes as an upgrade alternative and allows upgrading to an alpha/beta/release candidate version of Kubernetes. Default: false
> 
> allowRCUpgrades bool	Enable allowRCUpgrades will show release candidate versions of Kubernetes as an upgrade alternative and allows upgrading to a release candidate version of Kubernetes.

I can remove it in my next commit as it's probably better to enable it in a separate PR, and it seems a bit redundant with `allowExperimentalUpgrades`